dataland-frontend/src/components/pages/SearchTaxonomy.vue

```diff
@@ -2,47 +2,180 @@
   <AuthenticationWrapper>
     <TheHeader/>
     <TheContent>
-      <SearchTaxonomyHeader :scrolled="scrolled"/>
-      <EuTaxoSearchBar @autocomplete-focus="handleAutoCompleteFocus" :stockIndices="stockIndices" ref="euTaxoSearchBar"
-                       @scrolling="handleScrolling"/>
+      <SearchTaxonomyHeader :scrolled="pageScrolled"/>
+      <EuTaxoSearchBar v-if="!pageScrolled"
+                       ref="euTaxoSearchBar"
+                       :initial-input="currentInput"
+                       @searchBarInput="handleInput"
+                       @companyToQuery="handleCompanyQuery"/>
```

Comment:
Wäre es nicht sinnvoller, anstelle von initial-input und dem searchBarInput event (was auch nur eine lokale Variable up to date hält) in der EuTaxoSearchBar v-model zu implementieren und dann damit gegen eine lokale variable zu binden? (Oder funktioniert das hier aus irgendwelchen Gründen nicht?)

Reply:
Fair point, schau ich mir jetzt an. Danke für den Hinweis !